pulser-core/pulser/json/abstract_repr/deserializer.py

```diff
@@ -443,20 +454,33 @@ def deserialize_abstract_sequence(obj_str: str) -> Sequence:
         # This is kept for backwards compatibility
         seq.config_slm_mask(obj["slm_mask_targets"])
 
-    # Detuning Map configuration
+    to_config_det_map: dict[str, dict] = dict()
     if "dmm_channels" in obj:
-        for dmm_id, ser_det_map in obj["dmm_channels"]:
-            trap_coords = []
-            weights = []
-            for trap in ser_det_map["traps"]:
-                trap_coords.append((trap["x"], trap["y"]))
-                weights.append(trap["weight"])
-            det_map = DetuningMap(
-                trap_coordinates=trap_coords,
-                weights=weights,
-                slug=ser_det_map.get("slug"),
+        # Make a dictionnary with all the configured dmm channels
+        for dmm_name, ser_det_map in obj["dmm_channels"]:
+            splitted_dmm_name = dmm_name.split("_")  # dmm_id_num
+            dmm_id = "_".join(splitted_dmm_name[0:2])
+            call_num = (
+                0 if len(splitted_dmm_name) <= 2 else int(splitted_dmm_name[2])
```

Comment:
Okay I also felt that we should include the information on both the dmm_name and the dmm_id. It's just that since we decided to label the dmm_name as dmm_id_{number of time dmm has been declared} I figured I had all the information dmm_name...